tox.ini

```diff
@@ -60,4 +60,4 @@ extras =
 commands =
   python -c 'import shutil, pathlib; shutil.rmtree(pathlib.Path("docs") / "stubs", ignore_errors=True)'
   python -c 'import shutil, pathlib; shutil.rmtree(pathlib.Path("docs") / "_build" / "html" / ".doctrees", ignore_errors=True)'
-  sphinx-build -j auto -b html -W -T --keep-going {posargs} docs/ docs/_build/html
+  sphinx-build -j auto -W -T --keep-going {posargs} docs/ docs/_build/html
```

Comment:
I just looked this up.  It looks like `-b html` [is the default](https://www.sphinx-doc.org/en/master/man/sphinx-build.html).  Is there a reason it shouldn't be set explicitly?

Reply:
Exactly. The code will work just the same, so either approach is fine.

As a general philosophy, I bias much more towards not setting options that are the default. It's added noise. Defaults exist for a reason. When you rely on defaults, it makes it much clearer where you're diverging.

But either is fine and I can revert this if you have a preference.

Reply:
> It's added noise.

Modifying a line that is not necessary, with no explanation, is also added noise.  But the change is fine -- now that we've discussed it, let's keep it.

Reply:
For sure. For context, I was going to add an explanation once the PR was marked ready for review (it had `[wip]`). I review my own PRs before asking for reviews